src/context/mod.rs

```diff
@@ -377,24 +371,114 @@ impl<C: Blockchain> MatchstickInstanceContext<C> {
         if store.contains_key(&entity_type) && store.get(&entity_type).unwrap().contains_key(&id) {
             let entities = store.get(&entity_type).unwrap();
             let entity = entities.get(&id).unwrap().clone();
-            let entity = Entity::from(entity);
+            let entity: Vec<(Word, graph::prelude::Value)> = entity
+                .into_iter()
+                .map(|(k, v)| (Word::from(k), v))
+                .collect();
 
-            let res = asc_new(&mut self.wasm_ctx, &entity.sorted(), &GasCounter::new())?;
+            let res = asc_new(&mut self.wasm_ctx, &entity, &GasCounter::new())?;
```

Comment:
Maybe we should implement our version of `sorted()` https://github.com/graphprotocol/graph-node/blob/master/graph/src/data/store/mod.rs#L775 now that we don't use the Entity type